src/components/ToggleSwitch/ToggleSwitch.st.css

```diff
@@ -1,154 +1,75 @@
 :import {
     -st-from: "../../common/formatters.st";
-    -st-named: applyStyleMethod, applyOpacity;
+    -st-named: color, fallback, font, applyStyleMethod, applyOpacity;
```

Comment:
There's no need to use `applyStyleMethod` in this file.
You can just use: `color(color-3)`, etc.
`applyStyleMethod` was created only for edge cases where the `color` name is taken by a variable (or something like that)